test/integration/master/transformation_testcase.go

```diff
@@ -237,3 +239,31 @@ func (e *transformTest) readRawRecordFromETCD(path string) (*clientv3.GetRespons
 
 	return response, nil
 }
+
+func (e *transformTest) printMetrics() error {
```

Comment:
(aside) We should make a prometheus testing library so we don't need to keep reimplementing this...

Reply:
I agree.